CraftingRevisions/UserBlueprintData.cs

```diff
@@ -0,0 +1,178 @@
+﻿using CraftingRevisions.Exceptions;
+using MelonLoader.TinyJSON;
+
+namespace CraftingRevisions
+{
+	public class ModUserBlueprintData
+	{
+		/// <summary>
+		/// optional name, used for debugging
+		/// </summary>
+		public string? Name = null;
```

Comment:
Only ever initialize a field if you're not initializing to the default value for that data type (null or zero).